lib/src/utils.test.ts

```diff
@@ -5,7 +5,7 @@ describe('utils', () => {
   describe('pathToUrlString', () => {
     it('encode relative path like `pathToFileURL`', () => {
       const baseURL = pathToFileURL('').toString();
-      for (let i = 0; i < 128; i++) {
```

Comment:
This (temporarily) works around a Node regression documented in https://github.com/nodejs/node/issues/51167 and https://github.com/sass/embedded-host-node/pull/264